src/services/rfqm_service.ts

```diff
@@ -205,9 +205,15 @@ export class RfqmService {
     private static _getJobStatusFromSubmissions(submissionsMap: SubmissionsMap): RfqmJobStatus {
         // there should only be one mined transaction, which will either be successful or a revert
```

Comment:
by the time we get here, there should only be one mined tx and it should be confirmed